packages/grafana-ui/package.json

```diff
@@ -123,10 +120,27 @@
     "rollup-plugin-typescript2": "0.29.0",
     "rollup-plugin-visualizer": "4.2.0",
     "storybook-dark-mode": "1.0.8",
-    "ts-loader": "8.0.11",
     "typescript": "4.4.3",
     "webpack-filter-warnings-plugin": "1.2.1"
   },
+  "peerDependencies": {
```

Comment:
Do we know what sort of impact (if any) these packages moving into `peerDependencies` will have on plugin devs?

Reply:
Not sure, though CI passes so I guess it's not completely broken at least. In any case I think plugins are the textbook usecase for `peerDependencies` so I would hope it wouldn't cause any issues. I'll investigate a bit more carefully though.

Reply:
I'm thinking that depending on the package manager the dev is using those dependencies may not be installed now they are peer dependencies. I guess we can follow up on it once this PR is merged and the packages are published to see what impact there is and get fixes in place before 8.3 and/or document breaking changes.